ui/effects/use-last-visible-item.js

```diff
@@ -18,8 +24,11 @@ export default function useLastVisibleItem(injectedItem: ?InjectedItem, listRef:
       const timer = setTimeout(() => {
         if (listRef.current) {
           const screenBottom = window.innerHeight;
+
+          // claim preview tiles
```

Comment:
Can be list, tiles, or even non ClaimPreview entirely. It's self-documenting:  items = all items in the listRef